app/src/androidTest/java/com/andela/gkuti/mipista/MainActivityTest.java

```diff
@@ -0,0 +1,55 @@
+package com.andela.gkuti.mipista;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+import com.andela.gkuti.mipista.view.activity.MainActivity;
+import com.andela.gkuti.mipista.util.Date;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public class MainActivityTest extends ActivityInstrumentationTestCase2<MainActivity> {
+    public MainActivityTest() {
+        super(MainActivity.class);
+    }
+
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+    public void testOnCreate() throws Exception {
+        getActivity();
+        onView(withText("Mi Pista")).check(matches(isDisplayed()));
+        onView(withText("NOT TRACKING")).check(matches(isDisplayed()));
+    }
+//Test cannot be ran on a cloud emulator
```

Comment:
*Nitpick:* Can you please get rid of all the commented out code in your codebase.